src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriorsIntegrationTest.java

```diff
@@ -1,139 +1,90 @@
-* 8.6 Binding Effect; Headings. This Agreement shall be binding upon and inure to the benefit of the parties and their respective permitted successors and assigns. All headings are for convenience only and shall not affect the meaning of any provision of this Agreement.
-* 8.7 Governing Law. This Agreement shall be construed, governed, interpreted and applied in accordance with the internal laws of the Commonwealth of Massachusetts, U.S.A., without regard to conflict of laws principles.
-*/
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
 
-package org.broadinstitute.gatk.tools.walkers.variantutils;
-
-import org.broadinstitute.gatk.engine.walkers.WalkerTest;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
 import org.testng.annotations.Test;
 
-import java.util.Arrays;
+import java.io.IOException;
+import java.util.Collections;
+
+public final class CalculateGenotypePosteriorsIntegrationTest extends CommandLineProgramTest {
+
+    private final String largeDir = largeFileTestDir;
+    private final String dir= getToolTestDataDir();
 
-public class CalculateGenotypePosteriorsIntegrationTest extends WalkerTest {
+    private String CEUtrioFamilyFile = dir + "CEUtrio.ped";
+    private String threeMemberNonTrioFamilyFile = dir + "threeMemberNonTrio.ped";
 
-    private static String CEUtrioFamilyFile = privateTestDir + "CEUtrio.ped";
-    private static String CEUtrioTest = privateTestDir + "CEUtrioTest.vcf";
-    private static String CEUtrioPopPriorsTest = privateTestDir + "CEUtrioPopPriorsTest.vcf";
-    private static String threeMemberNonTrioFamilyFile = privateTestDir + "threeMemberNonTrio.ped";
-    private static String getThreeMemberNonTrioTest = privateTestDir + "threeMemberNonTrioTest.vcf";
+    //Note: these files were made by taking GATK3 files and changing the chromosome from 1 to 20
+    // (because our reference is on 20 and it does not matter for this tool anyway)
+    private String CEUtrioTest = dir + "CEUtrioTest_chr20.vcf";
+    private String CEUtrioPopPriorsTest = dir + "CEUtrioPopPriorsTest_chr20.vcf";
+    private String getThreeMemberNonTrioTest = dir + "threeMemberNonTrioTest_chr20.vcf";
 
-    @Test(enabled = true)
+    @Test
     //use the first 20 variants to save time; they have a nice range of AC from 4 to over 4000
-    public void testUsingDiscoveredAF() {
-        WalkerTestSpec spec = new WalkerTestSpec(
-                "-T CalculateGenotypePosteriors --no_cmdline_in_header" +
-                        " -o %s" +
-                        " -R " + b37KGReference +
+    public void testUsingDiscoveredAF() throws IOException {
```

Comment:
so the output is incorrect? I see PPs different from PLs in `expectedCGP_testUsingDiscoveredAF.vcf`, eg:
`GT:AD:DP:GQ:PL:PP       0/0:4,0:4:7:0,12,187:0,7,183`